apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -206,15 +273,67 @@ abstract class CC {
   }
 
   class ApiGetChat(val type: ChatType, val id: Long): CC() {
-    override val cmdString get() = "/_get chat ${CC.chatRef(type, id)} count=100"
+    override val cmdString get() = "/_get chat ${chatRef(type, id)} count=100"
     override val cmdType get() = "apiGetChat"
   }
 
   class ApiSendMessage(val type: ChatType, val id: Long, val mc: MsgContent): CC() {
-    override val cmdString get() = "/_send ${CC.chatRef(type, id)} ${mc.cmdString}"
+    override val cmdString get() = "/_send ${chatRef(type, id)} ${mc.cmdString}"
     override val cmdType get() = "apiGetChat"
   }
 
+  class AddContact: CC() {
+    override val cmdString get() = "/connect"
```

Comment:
Shouldn't this take an argument of the contact address to connect to?

Reply:
it's Connect that takes it - the counterpart to this